test/unit/cbus_hang.result

```diff
@@ -1,4 +0,0 @@
-	*** main ***
```

Comment:
👍 The same conclusion is for `cbus_hang.test` - it's not yet ready to be considered as `TAP-friendly` and it will be excluded from current patchset for later modifications.

Reply:
(Hehe! There is no single `is` in the `cbus_hang.c` and the single `ok` you were referring to, and I suspect it was inserted just for the purpose of providing `check_plan` anything to check against)